frontend/src/api/network/servingRuntimes.ts

```diff
@@ -22,17 +22,21 @@ const fetchServingRuntime = (
 ): Promise<ServingRuntimeKind> => {
   return getConfigMap(configNamespace, 'servingruntimes-config')
     .then((configmap) => {
-      const servingRuntime = YAML.parse(configmap.data?.['default-config'] || '');
-      if (servingRuntime === null) {
+      const overrideServingRuntime = YAML.parse(configmap.data?.['override-config'] || '');
```

Comment:
Is the whole point of this effort to retain the `default-config`?

Reply:
Not the author but i can confirm that this is the goal here, the current configuration will force the user to either select a default config or a custom one "loosing" the default configuration if they somehow want another spec, it makes sense to have a "overriding" config that you can add/remove whenever you want.

Reply:
Yes, that is the effort.  The deployer overwrites the default-config on upgrade and we want to keep our default runtime upgraded.  This would allow a savvy user to override it without changing the default.